### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by Bibin-Antony**

### CODE REVIEW

**Review by hemanshthakur — APPROVED** [review]
good to go

File: Container/CSS/list.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    font-size: 1.3rem;
+}
+.checkbox{
+    margin-right: 1rem;
+}
+.radio{
+    margin-right: 1rem;
+}
+
+input.checkbox,.radio{
```
Comment (ckahub) [inline Container/CSS/list.css:23]: follow principle of multiple of 4px (0.25rem) or 8px(0.5 rem ) , beacuse its good practice adv ised yesterday

**Review by ckahub — APPROVED** [review]
everything is good

File: Container/CSS/list.css (hunk 1 above)
Comment (Bibin-Antony) [reply Container/CSS/list.css:23]: will do

**Pull request merged**